cnf-certification-test/accesscontrol/suite.go

```diff
@@ -153,8 +152,11 @@ func TestSecConCapabilities(env *provider.TestEnvironment) {
 			}
 		}
 	}
-	tnf.ClaimFilePrintf("bad containers: %v", badContainers)
-	gomega.Expect(badContainers).To(gomega.BeNil())
```

Comment:
Although I prefer your approach (without gomega), it is worth noting that gomega assertions can also accept a text that will be printed only if the assertion fails. In this case it could be like this:
```
tnf.ClaimFilePrintf("bad containers: %v", badContainers)
gomega.Expect(badContainers).To(gomega.BeNil(), "Containers were found with incorrect security context capabilities.")
```